tests/template_test.py

```diff
@@ -24,33 +24,33 @@ def test_bake_project_default_adapter_name(cookies):
         assert  'dbt' in found_toplevel_files
         assert 'setup.py' in found_toplevel_files
         assert 'tox.ini' in found_toplevel_files
-        assert 'test' in found_toplevel_files
-    
+        assert 'tests' in found_toplevel_files
+
 def test_bake_direcotry_name(cookies):
     """bake and test against new words, if passes means its changing values in template"""
     with bake_in_temp_dir(cookies) as result:
         assert result.exit_code == 0
         assert result.exception is None
         assert result.context['directory_name'] == 'myadapter'
         assert result.context['directory_name'] != "hello_world"
-       
-def test_bake_deafult_adapter_src(cookies):
-    '''bake and test default version of adapter_src is sql'''
+
+def test_bake_deafult_is_sql_adapter(cookies):
+    '''bake and test default version of is_sql_adapter is true'''
     with bake_in_temp_dir(cookies) as result:
         assert result.exit_code == 0
         assert result.exception is None
-        assert result.context['adapter_src'] =='sql'
-        assert result.context['adapter_src'] != 'base'
-        assert result.context['adapter_src'] != '1'
-        assert result.context['adapter_src'] != '2'
+        assert result.context['is_sql_adapter'] =='true'
+        assert result.context['is_sql_adapter'] != 'false'
+        assert result.context['is_sql_adapter'] != '1'
+        assert result.context['is_sql_adapter'] != '2'
```

Comment:
I don't think we need to check the rest, since checking the first already confirmed what `context['is_sql_adapter']` is